.github/workflows/audit.yml

```diff
@@ -27,14 +27,15 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
+
 jobs:
   audit-rust:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: dtolnay/rust-toolchain@stable
-      - run: cargo install cargo-audit
-      - run: cargo audit
+      - uses: actions/checkout@v4
+      - uses: rustsec/audit-check@v1.4.1
```

Comment:
is there a reason not to use v2? It's just 3 days old so genuine question.

Reply:
I just copied from their README.md, didn't notice there was v2